content/en/docs/contribute/style/page-templates.md

```diff
@@ -0,0 +1,235 @@
+template to use for a new topic, start with the
+[concept template](#concept-template).
+{{< /note >}}
+
+{{% /capture %}}
+
+{{< toc >}}
+
+{{% capture body %}}
+
+## Concept template
+
+A concept page explains some aspect of Kubernetes. For example, a concept
+page might describe the Kubernetes Deployment object and explain the role it
+plays as an application is deployed, scaled, and updated. Typically, concept
+pages don't include sequences of steps, but instead provide links to tasks or
+tutorials.
+
+
+To write a new concept page, create a Markdown file in a subdirectory of the
+`/content/en/docs/concepts` directory, with the following characteristics:
+
+- In the page's YAML front-matter, set `content_template: templates/concept`.
+- In the page's body, set the following `capture` variables:
+
+    | Variable      | Required? |
+    |===============|===========|
+    | overview      | yes       |
+    | body          | yes       |
+    | whatsnext     | no        |
+
+    The page's body will look like this (without any optional sections you don't
+    need):
+    
+    ```
+    {% raw %}
+    {{% capture overview %}}
+    
+    {{% /capture %}}
+    
+    {{< toc >}}
+    
+    {{% capture body %}}
+    
+    {{% /capture %}}
+    
+    {{% capture whatsnext %}}
+    
+    {{% /capture %}}
+    {% endraw %}
+    ```
+
+- Within each section, write your content. Use the following guidelines:
+  - Use a minimum of H2 headings (with two leading `#` characters). The sections
+    themselves are titled automatically by the template.
+  - For `overview`, use a paragraph to set context for the entire topic.
+  - Add the `{{< toc >}}` shortcode to show an in-page table of contents.
+  - For `body`, explain the concept using free-form Markdown.
+  - For `whatsnext`, give a bullet list of up to 5 topics the reader might be
+    interested in reading next.
+
+An example of a published topic that uses the concept template is
+[Annotations](/docs/concepts/overview/working-with-objects/annotations/). The
+page you are currently reading also uses the content template.
+
+## Task template
+
+A task page shows how to do a single thing, typically by giving a short
+sequence of steps. Task pages have minimal explanation, but often provide links
+to conceptual topics that provide related background and knowledge.
+
+To write a new task page, create a Markdown file in a subdirectory of the
+`/content/en/docs/tasks` directory, with the following characteristics:
+
+- In the page's YAML front-matter, set `content_template: templates/task`.
+- In the page's body, set the following `capture` variables:
+
+    | Variable      | Required? |
+    |===============|===========|
+    | overview      | yes       |
+    | prerequisites | yes       |
+    | steps         | no        |
+    | discussion    | no        |
+    | whatsnext     | no        |
+
+    The page's body will look like this (without any optional sections you don't
```

Comment:
So are any of the sections below optional or are they mandatory? Its unclear to me

Reply:
I don't understand. The "Required" column in the table should cover this, I thought.